src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -119,12 +119,23 @@ public async Task<JObject> TryToRunOnLoadedClasses(string varName, CancellationT
                     }
                 }
                 var store = await proxy.LoadStore(sessionId, token);
+                var info = ctx.CallStack.FirstOrDefault().Method.Info;
```

Comment:
Should this be using `scopeId` to find the corresponding `Frame` from the callstack, and use the namespace from that? Eg. when you are not on the top frame.

If so, then tests for that are also needed.